tempo/serve/loader/env.py

```diff
@@ -2,34 +2,38 @@
 import uuid
 from subprocess import run
 from tempfile import NamedTemporaryFile
-from typing import Optional
+from typing import List, Optional
 
 import conda_pack
 import yaml
 
 from ...utils import logger
-from ..constants import MLServerEnvDeps
+from ..constants import MLServerEnvDeps, MLServerRuntimeEnvDeps
+from ..metadata import ModelFramework
 
 
-def _get_env(conda_env_file_path: str = None, env_name: str = None) -> dict:
+def _get_env(conda_env_file_path: str = None, env_name: str = None, platform: ModelFramework = None) -> dict:
```

Comment:
Can you explain this change?

Reply:
if we pack the conda environment, we need to add `mlserver` dependencies if not defined in `conda.yaml`

In the case of `mlflow` this is the case as the model `conda.yaml` generated will not have anything related to `mlserver`  as not required for training.

There is logic to add `mlserver` package in `_add_required_deps` however we also need the corresponding runtime to be packaged (e.g. `mlserver-mlflow` in this case) and it is dependant on the type of model used. A reference to the platform is able to let us do that.

There is logic that